pytest/tests/nearcore/fuzz_account_id.py

```diff
@@ -0,0 +1,11 @@
+import os
+import subprocess
+import sys
+
+sys.path.append('lib')
+
+import cargo_fuzz
+
+if __name__ == '__main__':
+    sys.exit(cargo_fuzz.run('core/account-id/fuzz', 'serde'))
+    sys.exit(cargo_fuzz.run('core/account-id/fuzz', 'borsh'))
```

Comment:
Actually, those need to be separate fuzz scripts.

Reply:
Right..